docs/book/installation/sylius_plus_installation.rst

```diff
@@ -40,7 +40,6 @@ Installing Sylius Plus as a plugin to a Sylius application
 .. code-block:: bash
 
     composer config repositories.plus composer https://sylius.repo.packagist.com/ShortNameOfYourOrganization/
-    composer config minimum-stability rc #due to the usage of some pre-stable packages (like SyliusRefundPlugin)
```

Comment:
Isn't it still needed due to InvoicingPlugin?

Reply:
Nope, because the latest version of InvoicingPlugin is [0.16.1](https://github.com/Sylius/InvoicingPlugin/releases/tag/v0.16.1) so there is no need to define the minimum stability for this plugin

Reply:
silly me, of course. We will have to come back to it, once we will start Invoicing stabilization